lib/KThemePlugin.js

```diff
@@ -39,13 +40,34 @@ import KCard from './KCard';
 import { themeTokens, themeBrand, themePalette, themeOutlineStyle } from './styles/theme';
 import globalThemeState from './styles/globalThemeState';
 
+import useKLiveRegion from './composables/useKLiveRegion';
+
+const { _mountLiveRegion } = useKLiveRegion();
+
 require('./grids/globalStyles.js'); // global grid styles
 
 /**
  * Install Kolibri theme helpers on all Vue instances.
  * Also, set up global state, listeners, and styles.
  */
 export default function KThemePlugin(Vue) {
+  // Note that if DOM live regions need to be demostrated
+  // on the KDS website, and therefore attached to the DOM,
+  // just call _mountLiveRegion() in the relevant documentation
+  // page's 'mounted' (see 'docs/pages/usekliveregio.vue' for an example)
+  if (!isNuxtServerSideRendering()) {
+    const onDomReady = () => {
+      _mountLiveRegion();
```

Comment:
I imagine we would add other features, such as inserting root element for teleporting, here to `onDomReady`